### PR TITLE
Add additional paths to customizeHLTIter0ToMkFit

### PR DESCRIPTION
**Pull request by Dr15Jones**
#### PR description:

The other paths also include hltIter0PFlowCkfTrackCandidates and need the change in order to be self-consistent.
The inconsistency was found by #34735
#### PR validation:

I ran workflow 11634.7 with this change and the changes from #34735 and it now runs fine.

### CODE REVIEW

**Comment by Dr15Jones** [comment]
please test

File: RecoTracker/MkFit/python/customizeHLTIter0ToMkFit.py [hunk 1]
```diff
@@ -68,12 +68,11 @@ def customizeHLTIter0ToMkFit(process):
...
     process.HLTDoLocalStripSequence += process.hltSiStripRecHits
-    process.HLTIterativeTrackingIteration0.replace(process.hltIter0PFlowCkfTrackCandidates,
-                                                   process.hltIter0PFlowCkfTrackCandidatesMkFitSiPixelHits +
-                                                   process.hltIter0PFlowCkfTrackCandidatesMkFitSiStripHits +
-                                                   process.hltIter0PFlowCkfTrackCandidatesMkFitEventOfHits +
-                                                   process.hltIter0PFlowCkfTrackCandidatesMkFitSeeds +
-                                                   process.hltIter0PFlowCkfTrackCandidatesMkFit +
-                                                   process.hltIter0PFlowCkfTrackCandidates)
+
+    replaceWith = process.hltIter0PFlowCkfTrackCandidatesMkFitSiPixelHits + process.hltIter0PFlowCkfTrackCandidatesMkFitSiStripHits + process.hltIter0PFlowCkfTrackCandidatesMkFitEventOfHits + process.hltIter0PFlowCkfTrackCandidatesMkFitSeeds + process.hltIter0PFlowCkfTrackCandidatesMkFit + process.hltIter0PFlowCkfTrackCandidates
```
Comment (slava77) [inline RecoTracker/MkFit/python/customizeHLTIter0ToMkFit.py:72]: please add linebreaks for readability (to fit in under 100 chars)

Comment (Dr15Jones) [reply RecoTracker/MkFit/python/customizeHLTIter0ToMkFit.py:72]: I tried that at first, but python does not allow line breaks for that expression

Comment (slava77) [reply RecoTracker/MkFit/python/customizeHLTIter0ToMkFit.py:72]: linebreaks are allowed inside braces.
wouldn't `replaceWith = ( ` work?

Comment (Dr15Jones) [reply RecoTracker/MkFit/python/customizeHLTIter0ToMkFit.py]: I was actually testing just that just before I headed out for lunch

**Comment by Dr15Jones** [comment]
code-checks

**Comment by slava77** [comment]
it looks like the bot is out for lunch or dinner

**Comment by slava77** [comment]
code-checks

in case mine works

**Comment by slava77** [comment]
> it looks like the bot is out for lunch or dinner

@smuzaffar 
please take a look, to get the bot to react to `code-checks`

**Comment by smuzaffar** [comment]
@slava77 , most of code check jobs run on lxplus nodes and for last few days we have issues with some of lxplus nodes where singularity can not run [a]. Unfortunately code checks for this PR was run 4 times on buggy nodes. 

Anyway, code checks are running now and results will be available soon

```
singularity -s exec /cvmfs/unpacked.cern.ch/registry.hub.docker.com/cmssw/cc7:amd64 sh -c 'export ORIGINAL_SINGULARITY_BIND=${SINGULARITY_BIND}; export SINGULARITY_BIND='\'''\'';  source /cvmfs/cms.cern.ch/cmsset_default.sh ; export PATH=$PATH:/usr/sbin;export PATH=$HOME/bin:$PATH; voms-proxy-init -voms cms -valid 24:00|| true ; cd /tmp/cmsbuild/jenkins/workspace/run-pr-code-checks; echo $PATH; /tmp/cmsbuild/jenkins/workspace/run-pr-code-checks/run.sh'
00:26:01 [91mERROR  : Failed to create user namespace: user namespace disabled
```

**Comment by Dr15Jones** [comment]
please test

**Comment by Dr15Jones** [comment]
@smuzaffar is the comparison test for this pr stuck in some endless loop? Seems to be doing the same thing over and over for the past 12 hours.

**Comment by smuzaffar** [comment]
@Dr15Jones not really, https://cmssdt.cern.ch/jenkins/job/compare-root-files-short-matrix/44643/ should finish in 10/20 mins. Normally it takes 2 hours when job starts.

**Comment by Dr15Jones** [comment]
@smuzaffar I checked this build before going to bed last night and the camparison had already been running for more than 2 hours. I then get up this morning and it still says it is running.

**Comment by smuzaffar** [comment]
Here is the sequence of jobs
- 12h ago started the pr build job https://cmssdt.cern.ch/jenkins/job/ib-run-pr-tests/17599/ 
- 12 h ago started the deployment of cvmfs https://cmssdt.cern.ch/jenkins/job/ib-run-pr-wait-deployment/11966/
- 5h ago started the RelVals job https://cmssdt.cern.ch/jenkins/job/ib-run-pr-relvals/8810/  (took 1h to finish)
- 2h ago started the Comparison job https://cmssdt.cern.ch/jenkins/job/compare-root-files-short-matrix/44643/

all look good in terms of jobs. As jenkins was very busy last night that is why there were delays beteween various jobs

**Comment by slava77** [comment]
+reconstruction

for https://github.com/cms-sw/cmssw/pull/34793	a297db5 
- code changes are in line with the PR description and the follow up review
- jenkins tests pass; the specific fail case targeted by this PR was tested as detailed in the PR description

**Comment by qliphy** [comment]
+1

**Pull request merged**